packages/core/src/components/portal/portal.tsx

```diff
@@ -106,18 +89,14 @@ export class Portal extends React.Component<PortalProps, PortalState> {
     }
 
     public componentWillUnmount() {
-        if (this.portalElement != null) {
-            this.portalElement.remove();
-        }
+        this.portalElement?.remove();
     }
 
-    private createContainerElement() {
+    private createPortalElement() {
         const container = document.createElement("div");
         container.classList.add(Classes.PORTAL);
         maybeAddClass(container.classList, this.props.className);
-        if (this.context != null) {
-            maybeAddClass(container.classList, this.context.blueprintPortalClassName);
-        }
+        maybeAddClass(container.classList, (this.context as PortalContextState)?.portalClassName);
```

Comment:
This is kind of awkward, but there's no better way around it with the new context API when used in class components. I guess React really wants you to use function components. We can remove this type cast when we refactor this component to be a function.